MAINTAINERS

```diff
@@ -0,0 +1,78 @@
+
+
+	List of maintainers and how to submit Bitcoin Core changes
+
+Please try to follow the guidelines below.  This will make things
+easier on the maintainers.  Not all of these guidelines matter for every
+trivial patch so apply some common sense.
+
+1.	Always _test_ your changes, however small.
+
+2.	Try to release a few ALPHA test versions to the net. Announce
+	them onto the bitcoin-dev channel and await results.
+
+3.	Make sure your changes compile correctly in multiple
+	configurations, and passes Travis.
+
+4.	When you are happy with a change make it generally available for
+	testing and await feedback.
+
+5.	Make a patch available to the relevant maintainer in github
+	or on the bitcoin-dev list.
+
+	PLEASE check your patch with the automated style checker
+	(clang-format) to catch trivial style violations.
+	See doc/developer-notes.md for guidance here.
+
+	PLEASE CC: the maintainers and mailing list on your changes.
```

Comment:
I don't think we need a mailing list post for every merge request... especially considering the ML is for Bitcoin, not Bitcoin Core.